joomla/joomla-cms
# [staging] Follow up on #23604 Custom modules editor field

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/23604#discussion_r252200030

### Summary of Changes
After that discussion and merge of #23604 @HLeithner and I thought that we may have the wrong code since 1.5 (see discussion above) concerning the way to get an editor field in Custom modules or similar using  `< customContent />` in their manifest.

This patch also simplifies the custom type module edit in frontend as we now check the xml, thus normalizing the code vs backend.

### Testing Instructions
Install [mod_customadv.zip](https://github.com/joomla/joomla-cms/files/2824136/mod_customadv.zip)

Create or edit a custom module and the module above in back-end and frontend to confirm the editor field displays.

@HLeithner 

Pinging also @johanjanssens (see discussion) as the original code comes from 1.5 and, before merging this, it would be important to know if there was a reason for it or if it was just a typo that made gently its way until now...
